### ISSUE TITLE
Epic: merge prep

### ISSUE THREAD

**Issue by wackerow** [ethereum/staking-launchpad#480]
Context: Epic from Q2 Roadmap https://github.com/ethereum/ethereum-org-website/issues/6161

## Epic Description
After a long wait, the Merge is almost upon us which brings some changes for new and existing stakers. This transition means the Staking Launchpad is in need of review to make sure all content is accurate and up-to-date for the staking community. 

A similar, broader initiative is being pursued in parallel to [update the ethereum.org resources](https://github.com/ethereum/ethereum-org-website/issues/6281). This epic will focus on stakers and node operators as the primary personas being targeted. 

## Involves 
- Scoping out what content changes, additions, or subtractions are required related to the Merge upgrade
- Preparing appropriate content changes
- Coordinating with external contributors to ensure accuracy
- Timely release to appropriately provide the staking community with accurate information about the Merge, with enough time to take actions necessary to help ensure a safe and seamless transition

## Goals
**Pre-Merge Prep**
- Merge readiness checklist, consolidating the most important information into a checklist for stakers
- Add notices to client selection pages to ensure everyone knows you need to run both a EL and CL client to sync the chain after the Merge
- Review and update FAQ for accuracy
- Add notice/reminder that the _testnet_ is still available to test your setup

**Post-merge, longer-term changes**
- Ensure references to proof-of-work are updated to either be historical references, or removed

## Checkpoints
(Not necessarily all in order)
- [x] Pre-merge content prepared for stakers and node operators, including "readiness checklist"
- [x] Pre-merge content implemented
- [x] Pre-merge content reviewed
- [ ] Full site content audit for merge-related content, organizing changes needed
- [ ] Post-merge content modified for accuracy
- [ ] Post-merge content reviewed

**Comment by samajammin** [comment]
#448 seems related to this 👍

**Comment by samajammin** [comment]
@wackerow similar to https://github.com/ethereum/ethereum-org-website/issues/6281 I think we can break out post-merge changes into a separate issue & close this one out. Please feel free to re-open if I missed anything or you disagree.
